consensus/src/dag/dag_state_sync.rs

```diff
@@ -218,7 +218,7 @@ impl DagStateSynchronizer {
             .get_signers_addresses(&self.epoch_state.verifier.get_ordered_account_addresses());
 
         match dag_fetcher
-            .fetch(request, responders, sync_dag_store.clone())
+            .fetch(request, responders, sync_dag_store.clone(), Box::pin(async {}))
```

Comment:
Will async {} return immediately in the select! inside?

Reply:
Correct, it's a placeholder. Next PR will fix it.